packages/connect/protocol-connect.js

```diff
@@ -12,5 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// workaround for bundlers that do not support subpath exports.
-export * from "./dist/esm/protocol-connect/index.js";
+/* eslint-disable */
+
+// Workaround for bundlers that do not support subpath exports.
+module.exports = require("./dist/cjs/protocol-connect/index.js");
```

Comment:
Why is this changed, but the other files are not?

Reply:
tbh, it's very confusing.  I think eslint is seeing these files as ESM and the `module.exports` is causing it to fail. If I change these files to have a `.cjs` extension, they pass. I borrowed the `.js` + `eslint-disable` from the Karma files I mentioned in the above comment.

Reply:
> I think eslint is seeing these files as ESM and the module.exports is causing it to fail. If I change these files to have a .cjs extension, they pass. 

The reason is that we run eslint from the root of the repository, where a package.json with type:module sits. This makes eslint believe all `.js` files should be ESM.

We can not change the file extension to `.cjs`. The correct fix would be to add an override for `.js` in this path. Maybe @dimitropoulos can help set this up? 

If that isn't possible, please update the linter comment to explain the situation.

Reply:
If you'd rather it be an override (instead of my use of `/* global` above) that works too. Totally cool if @dimitropoulos wants to set that up.

Reply:
ok: I updated it in dca59a36ebe1a286c2337d3878e6f606093070e2

For whatever it's worth, I personally would prefer to leave it as is because there's no feedbacking check that if these files are ever removed that this line from the eslint config will also be removed.  I don't mind that it's duplicated on a per-file basis because at least then the override can't be accidentally applied to other files that match the same pattern and also you can clearly see why it works the way it does just be seeing the one source file (instead of having to know there's a specific override just for these files in the root config).

But, it's all the same if that's what you guys think is best.  no problemo.

Reply:
I thought we would be able to set `sourceType: "commonjs"` for the offending files. I can't imagine that every CommonJS projects needs to set globals manually.  Does that not work?

Reply:
it does not work, yeah.  You still get `no-undef` triggering if you write the config like this:

```json
    {
      files: ["packages/connect/*.js"],
      parserOptions: {
        sourceType: "commonjs",
      },
    },
```

Reply:
Thanks for looking into it, @dimitropoulos.

Reply:
happy to help.. although... haha... I'm suddenly now quite curious why `sourceType` _doesn't_ work.  Feels like it should.  Oh well.  We have dozens of bigger fish to fry so I'll just hope to have the spontaneous "ah-ha" moment on that one in the future.